src/embedded.js

```diff
@@ -87,7 +87,7 @@ class Embedded {
     this._auth0profile = null // A user profile from Auth0
 
     // Set an initial UI Controller state for activation
-    this.state.setPanelClosed() // A default state of the panel is CLOSED
+    // this.state.setPanelClosed() // A default state of the panel is CLOSED
```

Comment:
why commented out?

Reply:
I have commented this to have panel opened on first startup, because there is no context menu with opening panel  and there is no opportunity to see the panel and set any options befor lookup any word.
It seems to me, that this way - it is much useful.
But it is not important for the task.

Reply:
ah, ok. this is why we need alpheios-project/components#274